### PR TITLE
Patient's Routes 

### PR DESCRIPTION
**Pull request by danielkapeleti**
I deleted the unnecessary files and the component. I renamed the pull request.

### CODE REVIEW

**Comment by nikoskleidis** [comment]
Daniel this is perfect. Before we merge this one, please make sure Prettier is up and running. I can see in some files that Prettier is not enabled to your machine. After you enable it, go through the 6 files you have added or modified in this PR and save them one more time in order to allow Prettier to run and format your code. Then commit and push again in order to update this PR

**Comment by danielkapeleti** [comment]
Hello Niko!

I pushed and made the PR which can be seen here:

https://github.com/SocialHackerCreteClass/FrodidApp-FrontEnd/compare/develop...danielkapeleti:master


I manually gave the command in the terminal prettier “filename” and I fixed the index.js in src/components/Menu

I believe that is what I can do at this point since Prettier requires some initiation after installation, that I am not yet able to configure.

Sincerely,

Daniel Kapeleti

Mobile: + 30 698 729 9370

**Comment by danielkapeleti** [comment]
Dear Niko,

After applying prettier, I pushed and made the PR  . However GitHub says there is nothing to compare:

https://github.com/SocialHackerCreteClass/FrodidApp-FrontEnd/compare/master...develop


Is it regarding prettier changes as not being changes at all ?

Sincerely,

Daniel Kapeleti

Mobile: + 30 698 729 9370

**Comment by nikoskleidis** [comment]
Please run Prettier in all 6 files

File: src/Pages/Patients/AllPatients/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from "react"
+import { useI18n } from "providers/I18n"
+
+AllPatients.propTypes = {}
+
+function AllPatients() {
+  const { t } = useI18n()
+  return (
+    <div>
+      <h1>{t("int.all-patients-header", "patients")}</h1>
```
Comment (nikoskleidis) [inline src/Pages/Patients/AllPatients/index.js:10]: This should produce a warning in the console. Open your DevTools and check it. 
Remove this warning by adding the `int.all-patients-header` key to the src/providers/I18n/translations.json for both en and el

**Pull request merged**